chapters/VK_EXT_debug_utils.txt

```diff
@@ -378,6 +385,14 @@ include::../api/protos/vkCreateDebugUtilsMessengerEXT.txt[]
 
 include::../validity/protos/vkCreateDebugUtilsMessengerEXT.txt[]
 
+[NOTE]
+.Note
+====
```

Comment:
Why as a Note? Specifying language should be outside a note IMO. Notes should only reiterate information that is already there but easy to miss from a formal description.